app/src/main/res/layout/context_menu.xml

```diff
@@ -5,23 +5,29 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
nit: `xmlns:` attrs should be at the top of the tag, above all the other attrs.

Reply:
fixed